editors/code/schemas/luaurc.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$id": "https://json.schemastore.org/luaurc.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
+        "languageMode": {
+            "type": "string",
+            "enum": ["strict", "nonstrict", "nocheck"]
+        },
+        "lintErrors": { "type": "boolean" },
+        "typeErrors": { "type": "boolean" },
+        "globals": {
+            "type": "array",
+            "items": { "type": "string", "pattern": "^[a-zA-Z_]\\w*" }
+        },
+        "aliases": {
+            "type": "object",
+            "additionalProperties": false,
+            "patternProperties": {
+                "^(?!\\.{1,2}$)(?!.*\\/)[a-zA-Z0-9\\-\\._]+$": {
```

Comment:
I don't completely get this regex. What is the purpose of the negative lookahead / lookbehind?

Reply:
to reject the string if is exacly "." or ".."